<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
<<<<<<< HEAD
   <bool>false</bool>
=======
   <bool>true</bool>
>>>>>>> d860567a9f431f0d31d092f380ceb9e0905486fc
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< HEAD
    <width>1185</width>
    <height>724</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1185</width>
    <height>724</height>
   </size>
  </property>
=======
    <width>1009</width>
    <height>557</height>
   </rect>
  </property>
>>>>>>> d860567a9f431f0d31d092f380ceb9e0905486fc
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
<<<<<<< HEAD
   <string notr="true">this-&gt;setStyleSheet(
    &quot;QLabel {&quot;
    &quot;  background: qlineargradient(x1:0, y1:0, x2:1, y2:1, &quot;
    &quot;              stop:0 rgb(58, 12, 163), &quot;
    &quot;              stop:0.5 rgb(98, 0, 238), &quot;
    &quot;              stop:1 rgb(144, 19, 254));&quot;
    &quot;  color: white;&quot;
    &quot;  font-size: 22px;&quot;
    &quot;  font-weight: bold;&quot;
    &quot;  padding: 10px 20px;&quot;
    &quot;  border: 2px solid rgb(255, 215, 0);&quot;
    &quot;  border-radius: 12px;&quot;
    &quot;  font-family: 'Segoe UI', 'Arial';&quot;
    &quot;  box-shadow: 0px 0px 15px rgba(255, 215, 0, 0.7);&quot;
    &quot;}&quot;
);
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="agentOne">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent; border: none;
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent; border: none;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="agentTwo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent; border: none;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
=======
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>851</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/header.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(237, 51, 59);
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
>>>>>>> d860567a9f431f0d31d092f380ceb9e0905486fc
 </widget>
 <resources/>
 <connections/>
</ui>
